napari/layers/_markers_layer/_constants.py

```diff
@@ -1,13 +1,24 @@
 from enum import Enum
 
+# Mapping of symbol alias names to the deduplicated name
+SYMBOL_ALIAS = {
```

Comment:
nice! since we have agreed to use enums going forward, these aliases should translate back to enums

also you should have two lines of whitespace between your import statements and your first line of code

Reply:
Good call. Done.